### ISSUE TITLE
Simple insert with transaction is not happening 

### ISSUE THREAD

**Issue by alwinjosh** [davidmoten/rxjava2-jdbc#13]
Trying to execute following insert query does not commit transaction due to transaction reference counter is not decremented.

db.update( "INSERT INTO ADDRESS_REL (ADDRESS_ID, ADDRESS_DETAILS  ) VALUES ( ?, ? )")
                .parameterStream(Flowable.just(105L,  "Address of 105"))
                .transacted()
                .counts()
                .blockingSubscribe(tx -> System.out.println(tx.isComplete() ? "complete" : tx.value()));

Do I missed any hint to transaction ?

**Comment by davidmoten** [comment]
Thanks for report, I'll have a look soon.

**Comment by davidmoten** [comment]
Yep, update/insert transactions need work. That case wasn't tested for because `transacted` is redundant in that situation. I'll see what I can do, plus I notice that the transactions generally needs more documentation.

**Comment by pkziel** [comment]
Is it still the problem ? Could you give an example how to make two updates/inserts in one transaction ?

**Comment by pkziel** [comment]
Is it still the problem ? Could you give an example how to make two updates/inserts in one transaction ?

**Comment by davidmoten** [comment]
oops, I'll revisit this

**Comment by yonghun8343** [comment]
i have a same problem... is this fixed???

**Comment by davidmoten** [comment]
Thanks for the reminder. I guess it's not fixed yet, I'll have a look soon.
